.github/workflows/trigger-circle-ci-workflow.yml

```diff
@@ -32,39 +32,20 @@ jobs:
     outputs:
       branch: ${{ env.branch }}
 
-  trigger-ci-tests:
+  trigger-normal-tests:
     runs-on: ubuntu-latest
     needs: get-branch
-    if: github.event_name == 'pull_request_target' && github.event.pull_request.draft == true && !contains(github.event.pull_request.labels.*.name, 'ci:pr') && !contains(github.event.pull_request.labels.*.name, 'ci:merged') && !contains(github.event.pull_request.labels.*.name, 'ci:daily')
+    if: github.event_name == 'pull_request_target' && contains(github.event.pull_request.labels.*.name, 'ci:normal')
```

Comment:
When we add label `ci:normal`, then a few seconds later add label `build` does it now trigger just once?

Reply:
How should this work? Isn't it up to GitHub to build a kind of throttling for firing events based on label changes if multiple labels are set at once? We could check whether a new workflow has started and cancel the previous ones. I don't know how straightforward this is, though.

Reply:
This feature is built into Actions, we already use it in the release workflow.

https://docs.github.com/en/actions/using-jobs/using-concurrency

I think it makes sense to add it here as well. It would probably be something like this at the top of [.github/workflows/trigger-circle-ci-workflow.yml](https://github.com/storybookjs/storybook/pull/23335/files/f30612849700a66f0d5b4da4d708bd4bfd59ef56#diff-84674a9f3aa27aa952c2e859e9899fb08b85139ff9476bc18efb849cec62d1af)

```
concurrency:
  group: ${{ github.workflow }}-${{ github.ref }}
  cancel-in-progress: true
```

The group tells GH that this run is grouped with any other run from the same workflow on the same branch. if we didn't set `cancel-in-progress: true` it would just hold them and run them sequentally.